packages/web-components/src/components/card/card.ts

```diff
@@ -15,6 +15,7 @@ import StableSelectorMixin from '../../globals/mixins/stable-selector';
 import DDSLink from '../link/link';
 import DDSCardFooter from './card-footer';
 import styles from './card.scss';
+import { PICTOGRAM_POSITION } from './defs';
```

Comment:
To align to `ICON_PLACEMENT` for `<dds-card-footer>`:

```suggestion
import { PICTOGRAM_PLACEMENT } from './defs';
```